### PR TITLE
update website/content/zh-cn/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md

### PR DESCRIPTION
**Pull request by sarazqy**
update website/content/zh-cn/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 1]
```diff
@@ -518,7 +518,7 @@ PATCH /apis/scheduling.k8s.io/v1/priorityclasses/{name}
 
   name of the PriorityClass
 -->
-- **name** （*路径参数*）: string，必须
+- **name** （**路径参数**）: string，必须
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:521]:
```suggestion
- **name** （**路径参数**）: string，必需
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
/assign

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 2]
```diff
@@ -613,9 +613,9 @@ Parameters
 
   name of the PriorityClass
 -->
-- **name** （*路径参数*）: string，必需
+- **name** （**路径参数**）: string，必需
 
-  PriorityClass 名称。
+  PriorityClass 名称
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:618]: 这个句号可以保留

**Review by Sea-n — COMMENTED** [review]
/lgtm

**Comment by mengjiao-liu** [comment]
/approve
Thanks for the catch!

**Pull request merged**